ydb/library/yql/minikql/comp_nodes/mkql_wide_combine.cpp

```diff
@@ -688,10 +696,12 @@ class TSpillingSupportState : public TComputationValue<TSpillingSupportState> {
     void SwitchMode(EOperatingMode mode) {
         switch(mode) {
             case EOperatingMode::InMemory: {
+                YQL_LOG(INFO) << "switching Memory mode to InMemory";
```

Comment:
And here also in Debug?

Reply:
Discussed that this log would be written once per task, so we decided to leave the INFO level.